pyramid/predicates.py

```diff
@@ -131,15 +131,17 @@ def __call__(self, context, request):
 
 class AcceptPredicate(object):
     def __init__(self, val, config):
-        self.val = val
+        if not is_nonstr_iter(val):
+            val = (val,)
```

Comment:
Why make this a tuple? Instead of a list or a set? I guess it doesn't matter...

Reply:
there's no difference for this purpose and I like immutable objects